src/db/models/Article.js

```diff
@@ -32,11 +32,11 @@ export default (sequelize, DataTypes) => {
       subscriptionType: {
         type: DataTypes.ENUM('free', 'premium'),
         field: 'subscription_type',
-        default: 'free',
+        defaultValue: 'free',
       },
       status: {
         type: DataTypes.ENUM('published', 'draft', 'trashed', 'banned'),
```

Comment:
Let create the enum values as constants in util/constants as 
```javascript
export const PUBLISHED = 'PUBLISHED'; // etc
```

Reply:
This has been resolved
It has been exported into 
```javascript
src/routes/utils/constants.js
```